packages/bundler/provider.ts

```diff
@@ -45,24 +46,36 @@ export class BundlerProvider {
   /* rpc */
 
   public async getChainId(): Promise<bigint> {
-    const chainId = await this.bundler.send<HexString>({
-      method: BundlerRpcMethod.eth_chainId
-    })
-    return number.toBigInt(chainId)
+    try {
+      const chainId = await this.bundler.send<HexString>({
+        method: BundlerRpcMethod.eth_chainId
+      })
+      return number.toBigInt(chainId)
+    } catch (err) {
+      throw err
+    }
```

Comment:
If catch block only re-throw the error out, we can just skip this try/catch.

Reply:
[remove try catch since only re-throw the error](https://github.com/pilagod/waallet-extension/pull/179/commits/ec8019568aff2bf3edb5c338b8ae9fe45f0b3417)